elevenlabs_mcp/server.py

```
@@ -0,0 +1,900 @@
+    """
+)
+@_safe_api
+def create_agent(
+    name: str,
+    first_message: str,
+    system_prompt: str,
+    voice_id: str = DEFAULT_VOICE_ID,
+    language: str = "en",
+    llm: str = "gemini-2.0-flash-001",
+    temperature: float = 0.5,
+    max_tokens: int = None,
```

Ruff (0.15.0)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)